util/uploadTemplate.sh

```diff
@@ -103,7 +103,6 @@ main() {
     sed -i "s#.*aws-parallelcluster.*/templates/raid-substack-\${version}.cfn.json.*#\"https://${_s3_folder_url}/raid-substack.cfn.json\",#" ${_temp_dir}/aws-parallelcluster.cfn.json
     sed -i "s#.*aws-parallelcluster.*/templates/efs-substack-\${version}.cfn.json.*#\"https://${_s3_folder_url}/efs-substack.cfn.json\",#" ${_temp_dir}/aws-parallelcluster.cfn.json
     sed -i "s#.*aws-parallelcluster.*/templates/fsx-substack-\${version}.cfn.json.*#\"https://${_s3_folder_url}/fsx-substack.cfn.json\",#" ${_temp_dir}/aws-parallelcluster.cfn.json
-    sed -i "s#.*aws-parallelcluster.*/templates/batch-substack-\${version}.cfn.json.*#\"https://${_s3_folder_url}/batch-substack.cfn.json\",#" ${_temp_dir}/aws-parallelcluster.cfn.json
```

Comment:
This must be restored.
It is required if we want to use a custom template during development process.